plugins/PrivacyManager/vue/src/ManageGdpr/ManageGdpr.vue

```diff
@@ -43,6 +43,7 @@
                 :show-all-sites-item="true"
                 :switch-site-on-select="false"
                 :show-selected-site="true"
+                @update:modelValue="changeSite($event)"
```

Comment:
Changing the site will now trigger a method to check if the profile is enabled. With that we can decide if we can suggest the userId segment or use the visitId segment instead.